### ISSUE TITLE
New Psy Shell v0.10.9 throws error when start

### ISSUE THREAD

**Issue by MiRacLe-RPZ** [bobthecow/psysh#686]
```
$ cd bin; curl -s -L 'https://github.com/bobthecow/psysh/releases/download/v0.10.9/psysh-v0.10.9.tar.gz' | tar xvz; chmod +x psysh; cd ..
psysh
$ bin/psysh -v 
PHP Fatal error:  Uncaught Error: Call to undefined function str_starts_with() in phar:///bin/psysh/vendor/symfony/console/Input/InputOption.php:21
Stack trace:
#0 phar:///bin/psysh/src/Configuration.php(185): _HumbugBox58367621d966\Symfony\Component\Console\Input\InputOption->__construct('config', 'c', 2, 'Use an alternat...')
#1 phar:///bin/psysh/src/functions.php(193): Psy\Configuration::getInputOptions()
#2 /bin/psysh(155): Psy\{closure}()
#3 {main}
  thrown in phar:///bin/psysh/vendor/symfony/console/Input/InputOption.php on line 21
```

Works fine with previous release

```
$ cd bin; curl -s -L 'https://github.com/bobthecow/psysh/releases/download/v0.10.8/psysh-v0.10.8.tar.gz' | tar xvz; chmod +x psysh; cd ..
psysh
$ bin/psysh -v
Psy Shell v0.10.8 (PHP 7.4.24 — cli) by Justin Hileman
New version is available (current: v0.10.8, latest: v0.10.9)
>>> 
```

**Comment by bobthecow** [comment]
Hmmm. That's a PHP 8 only function, and shouldn't be called in the pre-built binaries. I'll take a look. In the meantime, the `compat` build might work for you.

**Comment by bobthecow** [comment]
Weird that smoke tests passed for this release in both PHP7 and PHP5.

![image](https://user-images.githubusercontent.com/53660/137317780-f0096e99-5ba0-439d-aafe-79c38a42ecc6.png)
![image](https://user-images.githubusercontent.com/53660/137317817-9deb2ff8-484b-4e40-abeb-ed2aef5d6113.png)

**Comment by bobthecow** [comment]
This has been resolved in v0.10.11.
